src/pages/Shop/index.js

```diff
@@ -224,43 +218,50 @@ const listDropDownView = [MIN_VIEW_SHOP, AVERAGE_VIEW_SHOP, MAX_VIEW_SHOP];
 
 const listDropDownSort = ["desc", "asc"];
 
-const DropDown = ({
-  getListProducts,
-  dropMenuIsOpen,
+const DropDownHeader = ({
+  title,
   arr,
-  changeParamsValue,
-  changeParamsName,
-  params
+  changeDropDownState,
+  value,
+  visible,
+  onClick
 }) => {
+  return (
+    <div className={styles.dropMenu} onClick={changeDropDownState}>
+      <h4 className={styles.dropMenuTitle}>{title}</h4>
+      <div className={styles.dropMenuQty}>
+        <DropDown arr={arr} visible={visible} onClick={onClick} />
+      </div>
+      <p className={styles.valueDropDown}>{value}</p>
+      <div
+        className={
+          visible
+            ? `${styles.dropMenuControl} ${styles.dropMenuControlOpen}`
+            : `${styles.dropMenuControl}`
+        }
+      >
+        <TriangleTop />
+      </div>
+    </div>
+  );
+};
+
+const DropDown = ({ visible, arr, onClick }) => {
   return (
     <div
       className={
-        !dropMenuIsOpen
-          ? `${styles.dropMenuListClose}`
-          : `${styles.dropMenuList}`
+        !visible ? `${styles.dropMenuListClose}` : `${styles.dropMenuList}`
       }
     >
       {arr.map((item, index) => (
         <p
           className={styles.dropMenuItem}
-          onClick={() =>
-            getListProducts(
-              Object.assign(
-                params,
-                changeParamsName === "_sort" && {
-                  _sort: `${changeParamsValue}${item}`
-                },
-                changeParamsName === "_limit" && {
-                  _limit: `${changeParamsValue}${item}`
-                }
-              )
-            )
-          }
+          onClick={() => onClick(item)}
           key={index}
         >
           <span
             className={
-              dropMenuIsOpen
+              visible
                 ? `${styles.closeMenuActive}`
                 : `${styles.closeMenuUnactive}`
```

Comment:
the worst use css modules I ever seen!!!

you really need  such phrase `${styles.closeMenuActive}`

or you just use
```
visible
  ? styles.closeMenuActive
  : styles.closeMenuUnactive
```